helix-term/src/ui/editor.rs

```diff
@@ -229,6 +235,7 @@ impl EditorView {
 
         if config.inline_diagnostics.disabled()
             && config.end_of_line_diagnostics == DiagnosticFilter::Disable
+            && config.enable_diagnostics
```

Comment:
minute nitpick but since `config.enable_diagnostics` will override the other 2, it'll be better if it comes first.

Reply:
Okay